support/docker/production/config/custom-environment-variables.yaml

```
@@ -19,6 +19,7 @@ database:
   suffix: "PEERTUBE_DB_SUFFIX"
   username: "PEERTUBE_DB_USERNAME"
   password: "PEERTUBE_DB_PASSWORD"
+  ssl: "PEERTUBE_DB_SSL"
```

```suggestion
  ssl:
  __name: "PEERTUBE_DB_SSL"
  __format: "json"
```

So it's interpreted as a boolean